### PR TITLE
[opus] Remove cpp_info.name

### PR DESCRIPTION
**Pull request by danimtb**
Specify library name and version:  **opus/all**

Related to conan-io/hooks#142

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/opus/all/conanfile.py [hunk 1]
```diff
@@ -62,11 +62,12 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
 
     def package_info(self):
+        self.cpp_info.names['cmake_find_package'] = 'OPUS'
+        self.cpp_info.names['cmake_find_package_multi'] = 'OPUS'
+        self.cpp_info.names['pkg_config'] = 'opus'
```
Comment (jgsogo) [inline recipes/opus/all/conanfile.py]: Not needed, right? It is the name of the package

```suggestion
```

File: recipes/opus/all/conanfile.py [hunk 2]
```diff
@@ -62,11 +62,11 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
 
     def package_info(self):
+        self.cpp_info.names['cmake_find_package'] = 'OPUS'
```
Comment (uilianries) [inline recipes/opus/all/conanfile.py]:
```suggestion
        self.cpp_info.names['cmake_find_package'] = 'Opus'
```

Comment (uilianries) [reply recipes/opus/all/conanfile.py]: The cmake files provided by opus project are CamelCase, not UPPERCASE:

```
-- Install configuration: "Release"
-- Installing: lib/libopus.a
-- Installing: include/opus/opus.h
-- Installing: include/opus/opus_custom.h
-- Installing: include/opus/opus_defines.h
-- Installing: include/opus/opus_multistream.h
-- Installing: include/opus/opus_projection.h
-- Installing: include/opus/opus_types.h
-- Installing: lib/pkgconfig/opus.pc
-- Installing: lib/cmake/Opus/OpusTargets.cmake
-- Installing: lib/cmake/Opus/OpusTargets-release.cmake
-- Installing: lib/cmake/Opus/OpusConfig.cmake
-- Installing: lib/cmake/Opus/OpusConfigVersion.cmake
```

File: recipes/opus/all/conanfile.py [hunk 3]
```diff
@@ -62,11 +62,11 @@ def package(self):
...
+        self.cpp_info.names['cmake_find_package_multi'] = 'OPUS'
```
Comment (uilianries) [inline recipes/opus/all/conanfile.py]:
```suggestion
        self.cpp_info.names['cmake_find_package_multi'] = 'Opus'
```
ditto

File: recipes/opus/all/conanfile.py (hunk 2 above)
Comment (danimtb) [reply recipes/opus/all/conanfile.py]: Ok, I kept the same name but did not check it correctness. Thanks for checking

**Pull request merged**